istioctl/pkg/waypoint/waypoint.go

```diff
@@ -87,6 +89,34 @@ func Cmd(ctx cli.Context) *cobra.Command {
 				}},
 			},
 		}
+		// Determine which traffic address type to apply the waypoint to, if none
+		// then default to "service" as the waypoint-for traffic address type.
+		validAddressTypes := map[string]bool{
+			"service":  true,
```

Comment:
nit: it's only 4 bits, but stylistically, this is better expressed with a `struct{}` as the value type:
```suggestion
			"service":  struct{}{},
```

Reply:
Good point. Probably `pkg/util/sets/set.go` would be _most idiomatic_ for the project.

Reply:
or `pkg/sets`